src/Data/Form.purs

```diff
@@ -247,6 +255,11 @@ eval get page = do
       default <- traverse (evalExpr get) input.default
       value <- traverse (evalExpr get) input.value
       pure (Toggle { default, value })
+    Typeahead input -> do
+      default <- traverse (evalExpr get) input.default
```

Comment:
Having a `default` for typeaheads poses some interesting questions. Can we ensure it has the same shape as API data? Do we need to?

Reply:
No idea. I was going off the Notion and adding as few things as possible. More than happy to remove it if it makes things simpler.

Reply:
I was just thinking out loud. I'm okay to keep it until it actually proves problematic

Reply:
Ah. Gotcha!